### PR TITLE
Final touches branch

### PR DESCRIPTION
**Pull request by Kwaleyela-Ikafa**
## In this branch we:

- Used the involvement API to post and fetch the number of likes and comments
- Used an API of our choice to generate quotes to be displayed in the UI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,86 @@
-# webpack-exercise
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Sigma-Quotes
+
+> In this project we've created a dynamic website using two seperate API's to display a list of some quotes along with the ability to like them and leave comments in a dynamically loaded stylish popup.
+
+[Project Video](https://drive.google.com/file/d/1Yj6dQN7wGpV2p7y_UtH5XdKgyHX4TVCI/view?usp=sharing)
```
Comment (leonardodiasb) [inline README.md:7]: - [ ] Great video presentation. The project requirements require highlighting a piece of code or something you are proud of. I see that you mention a function that checks how many cards are being displayed, kindly try to show the code of this function or the background which changes the color from time to time. Also, try to be more professional. Introducing yourselves is a good way to start a presentation.

image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Kwaleyela-Ikafa, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Nice visuals. ✔️ 
 - Professional README. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

 - [NOTE] The Microverse group project workflow requires that while you are adding new features, you work on 'feature-branches' and open pull requests from 'feature-branch' to 'dev' for you to review each others' code, and when you finish the project, the 'dev' branch will be finished and you can open a pull request from 'dev' to 'main' and thus request the review to an external code reviewer. This is a case for invalidating the pull request, but as the previous code reviewer didn't mention this, I should proceed with the review. Please read [this article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/code_review_flow_group_projects.md) carefully and make sure you understand so in the next projects you make use of Gitflow properly.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Kwaleyela-Ikafa) [reply README.md]: we will redo the video but we will not be creating a new pull request as you stated it is optional and we have already begun the review  process

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Kwaleyela-Ikafa , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

### Great video presentation 👍🏻 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**